### PR TITLE
Paying for College - Non-first-year student features

### PR DESCRIPTION
**Pull request by mistergone**
This Pull Request adds features for non-first-year students. It also reformats some graphs and content.

## Changes
- Changes to school selection screen which allows a student to select their previous schooling
- Changes to debt calculation and inputs for current student debt
- Updates to graphs and content to improve design and clarity

## How to test this PR
1.  Pull down the branch and get it running
2. Check the various fields and user experiences of non-first-year students (see notes below)

## Notes and todos
- For details on how the user experience should work, please consult internal issues 356, 357, and 359.

## Checklist
- [x] PR has an informative and human-readable title
- [x] Changes are limited to a single goal (no scope creep)
- [x] Code follows the standards laid out in the [CFPB development guidelines](https://github.com/cfpb/development)
- [x] Future todos are captured in comments and/or tickets

#### Browser testing
Visually tested in the following supported browsers:
- [x] Firefox
- [x] Chrome
- [x] Safari
- [x] Edge 18 (the last Edge prior to it switching to Chromium)
- [x] Internet Explorer 11 and 8 (via emulation in 11's dev tools)
- [x] Safari on iOS
- [x] Chrome on Android

#### Accessibility
- [x] Keyboard friendly (navigable with tab, space, enter, arrow keys, etc.)
- [x] Screen reader friendly
- [x] Does not introduce new errors or warnings in [WAVE](https://wave.webaim.org/extension/)

#### Other
- [x] Does not introduce new lint warnings
- [x] Flexible from small to large screens

### CODE REVIEW

File: cfgov/unprocessed/apps/paying-for-college/js/dispatchers/update-models.js [hunk 1]
```diff
@@ -6,6 +6,7 @@ import { financialModel } from '../models/financial-model.js';
 import { financialView } from '../views/financial-view.js';
 import { getStateByCode } from '../util/other-utils.js';
 import { getSchoolData } from '../dispatchers/get-api-values.js';
+import { navigationView } from '../views/navigation-view.js';
```
Comment (virginiacc) [inline cfgov/unprocessed/apps/paying-for-college/js/dispatchers/update-models.js:9]: Is this import necessary?

File: cfgov/unprocessed/apps/paying-for-college/js/models/constants-model.js [hunk 2]
```diff
@@ -21,7 +21,8 @@ const constantsModel = {
       yearOne: 0,
       yearTwo: 0,
       yearThree: 0
-    }
+    },
+    existingDebtRate: 0.07
```
Comment (virginiacc) [inline cfgov/unprocessed/apps/paying-for-college/js/models/constants-model.js:25]: Probably doesn't need to be addressed immediately, but should this be managed in the django admin?

File: cfgov/paying_for_college/jinja2/paying-for-college/college-cost-blocks/02-costs.html [hunk 3]
```diff
@@ -84,52 +104,63 @@ <h3 class="u-border-bottom">Indirect costs</h3>
...
-                        <li>Find out if this school offers block tuition (also known as flat rate tuition) &mdash; when it costs the same to take anywhere from 12 to 18 credits, for example.</li>
-                        <li data-state-based-visibility="costs-question-yes">If your costs will be higher than the school predicts, contact the financial aid office. You may be eligible for more aid. </li>
-                </ul>' | safe
+                '',
+                'If your actual costs will be higher than the estimates from the school, talk to the financial aid office. You may be able to appeal for more aid.' | safe
             ) }}
+            <ul>
+                <li>You may also qualify for additional aid if you will be paying for health insurance, childcare, a personal computer, disability support, or study abroad.</li>
+                <li>Just because you qualify for more aid doesn’t mean you will receive it. And remember, financial aid includes student loans. More financial aid can mean an offer to borrow more money.</li>
+            </ul>
```
Comment (virginiacc) [inline cfgov/paying_for_college/jinja2/paying-for-college/college-cost-blocks/02-costs.html:122]: I wonder if the above content could / should be in a notification block? That seems to be the method we generally use when interspersing guidance with form fields...

File: cfgov/unprocessed/apps/paying-for-college/js/dispatchers/update-models.js (hunk 1 above)
Comment (mistergone) [reply cfgov/unprocessed/apps/paying-for-college/js/dispatchers/update-models.js:9]: Nope! I think it's a remnant of the "skip the intro page" work I was doing. I've removed it.

File: cfgov/unprocessed/apps/paying-for-college/js/models/constants-model.js (hunk 2 above)
Comment (mistergone) [reply cfgov/unprocessed/apps/paying-for-college/js/models/constants-model.js:25]: Yes, it should. This is a stand-in, and we haven't discussed much what this value should be. We can do a fast-follow of adding it to the django admin.

File: cfgov/paying_for_college/jinja2/paying-for-college/college-cost-blocks/02-costs.html (hunk 3 above)
Comment (mistergone) [reply cfgov/paying_for_college/jinja2/paying-for-college/college-cost-blocks/02-costs.html:122]: Sure, we can ask the team what they think. Easy change to make, if we want to.

Comment (virginiacc) [reply cfgov/paying_for_college/jinja2/paying-for-college/college-cost-blocks/02-costs.html:122]: Maybe we can add this to the general hierarchy updates work

**Review by virginiacc — APPROVED** [review]

**Pull request merged**